app/views/messages/_messages_table.html.erb

```diff
@@ -1,4 +1,4 @@
-<table class="table table-sm align-middle">
+<table class="table table-sm align-middle messages-table">
```

Comment:
Personally I'd prefer to use a data-attribute or `js-`-prefixed CSS selector when defining JS-hooks, but I wanted to keep this consistent with other code I read.